File: 1-chapter/Part_2/exponentByIteration.js
function exponentByIteration(a, n){
    let result = 1
    for (let i = 0; i < n; i++){
        result *= a;
    }
    return result
}

console.log(exponentByIteration(3, 6))
console.log(exponentByIteration(10, 8))
console.log(exponentByIteration(17, 10))

File: 1-chapter/Part_2/exponentByIteration.py
def exponentByIteration(base, exponent):
    result  = 1
    for i in range(exponent):
        result *= base
    return result

print(exponentByIteration(2, 2))
print(exponentByIteration(10, 3))
print(exponentByIteration(17, 10))